.github/workflows/link-check-deployment-status.yml

```diff
@@ -17,6 +17,10 @@ on:
         required: false
         description: "The file the link check report will be written to before publishing"
         default: "link-check-report.md"
+      config:
+        type: 'string'
+        description: 'The path to the link-check config file'
+        default: 'config/link-check/config.yml'
```

Comment:
Saw this was hard-coded in this workflow while the all workflow is configurable, copied here